android-components/components/browser/engine-gecko/src/main/java/mozilla/components/browser/engine/gecko/GeckoEngine.kt

```diff
@@ -399,7 +410,6 @@ class GeckoEngine(
         runtime.webExtensionController.enable((extension as GeckoWebExtension).nativeExtension, source.id).then(
             {
                 val enabledExtension = GeckoWebExtension(it!!, runtime)
-                webExtensionDelegate?.onEnabled(enabledExtension)
```

Comment:
Are we removing this from here, as we will get a respective call on `WebExtensionController.AddonManagerDelegate.onDisabled` and `WebExtensionController.AddonManagerDelegate.onEnabled`?

Reply:
By reading the test I confirmed this!

Reply:
Yes, we'd call `onDisabled()` or `onEnabled()` twice otherwise.

Reply:
👍🏽